src/mon/MonmapMonitor.cc

```diff
@@ -173,8 +173,9 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
 
   if (prefix == "mon stat") {
     mon->monmap->print_summary(ss);
-    ss << ", election epoch " << mon->get_epoch() << ", quorum " << mon->get_quorum()
-       << " " << mon->get_quorum_names();
+    ss << ", election epoch " << mon->get_epoch() << ", leader "
```

Comment:
@tchaikov  :smile: Yes that is true.But for the people who do not familiar with paxos, he or she may not know that.So make it more clear here for user.